package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "heroku-redis",
+  "name": "@heroku-cli/plugin-redis",
   "description": "Heroku plugin to manage Heroku Redis",
-  "version": "1.3.0",
+  "version": "1.3.2",
```

Comment:
Can we do version bumps separately via `np`? I'm happy to do it myself or fill in anything that's missing in the docs.

Reply:
I wanted to do that originally but I think on this PR I messed up the first couple of releases so I had to change the version number. 1.3.2 is already released, so if you wanted to release a new one you would have to release 1.3.3. We're happy to change it back to 1.3.0 though if that's what you would prefer

Reply:
Ok that's fine then.